oe_theme.theme

```diff
@@ -1247,10 +1247,9 @@ function oe_theme_preprocess_block__system_branding_block(&$variables) {
   if (!EuropeanUnionLanguages::hasLanguage($language_code)) {
     $language_icon_mapping = [
       'nb' => 'no',
-      'tr' => 'tu',
     ];
     if (isset($language_icon_mapping[$language_code])) {
-      // Set correct icon for Norwegian and Turkish languages.
+      // Set correct icon for Norwegian languages.
```

Comment:
Do we really need `$language_icon_mapping` array now? Can we just check if the langcode is `nb` and change it to `no`?

Reply:
fixed